components/eamxx/docs/user/coarse_nudging.md

```diff
@@ -5,21 +5,23 @@ Instead, in EAMxx, it is possible to nudge from coarse data.
 This is done by remapping the coarse data provided by the user to the runtime physics grid of EAMxx.
```

Comment:
I'm kind of weirded out by calling the file name "coarse_nudging", which always makes me wonder if it is something different than what I think of as nudging. Like maybe it is something called coarse that is your nudging target. Could you get rid of the coarse mentions except having a sentence saying that right now the only data you can nudge to is at either the same or coarser resolution than the model grid?